var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Podio(CMakePackage):
+    """PODIO, or plain-old-data I/O, is a C++ library to support the creation
+    and handling of data models in particle physics."""
+
+    homepage = "https://github.com/AIDASOFT/podio"
+    url      = "https://github.com/AIDASoft/podio/archive/v00-09-02.tar.gz"
+    git      = "https://github.com/AIDADSOFT/podio.git"
+
+    version('00-09-02', sha256='8234d1b9636029124235ef81199a1220968dcc7fdaeab81cdc96a47af332d240',  extension='tar.gz', preferred=True)
+    version('00-09', sha256='3cde67556b6b76fd2d004adfaa3b3b6173a110c0c209792bfdb5f9353e21076f')
+    version('00-08', sha256='9d035a7f5ebfae5279a17405003206853271af692f762e2bac8e73825f2af327')
+    version('master', branch='master')
+
+    variant('build_type', default='Release',
+            description='The build type to build',
+            values=('Debug', 'Release'))
+
+    variant('cxxstd',
+            default='17',
+            values=('14', '17'),
+            multi=False,
+            description='Use the specified C++ standard when building.')
+
+    depends_on('cmake', type='build')
+    depends_on('root@6.08.06:')
```

Comment:
I think the `depends_on` for `root` and `tbb` will need to account for the `cxxstd` variant to ensure ABI matching. To my knowledge there's no automatic way to do this, but the following is the easiest I've seen:

```python
_cxxstd_values=(14, 17)
variant('cxxstd',
        default='17',
        values=('14', '17'),
        multi=False,
        description='Use the specified C++ standard when building.')

for s in _cxxstd_values:
    depends_on('root@6.08.06: cxxstd=' + s, when='cxxstd=' + s)
    depends_on('tbb cxxstd=' + s, when='cxxstd='+s)